### ISSUE TITLE
Type errors when initializing (Typescript)

### ISSUE THREAD

**Issue by ColeTownsend** [pusher/push-notifications-web#67]
Hey team!

I know you added TS typings to this library which is helpful. When following the [set up guide](https://pusher.com/docs/beams/getting-started/web/sdk-integration) I'm getting the following error:

`Property 'Client' does not exist on type 'typeof import("@pusher/push-notifications-web")'`

### My code

```
import * as PusherPushNotifications from "@pusher/push-notifications-web";

const beamsClient: PusherPushNotifications.PushNotificationsInstance = new PusherPushNotifications.Client(
  {
    instanceId: "f5841e55-2273-4489-8cd0-71f799dc7854",
  },
);
```

**Comment by henrycheung19** [comment]
I also come across exactly the same situation.

would there be npm i @types/pusher__push-notifications-web ?

**Comment by jonathanlloyd** [comment]
Hi! Thanks for the report :slightly_smiling_face: 
I've shipped a fix in 1.0.3 which should solve this issue

**Comment by ColeTownsend** [comment]
Thanks @jonathanlloyd!
